.vscode/settings.json

```diff
@@ -20,6 +20,7 @@
     "ulogic",
     "uninstantiated",
     "Uniquify",
+    "useclauses",
```

Comment:
I think use clause is two words and should be in code always as use_clause or useClause thus this should not be here